Services/Services.Application/Repositories.Abstractions/IDoctorRepository.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Services.Domain.Entities;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Services.Application.Repositories.Abstractions
+{
+    public interface IDoctorRepository : IGeneralRepository<Doctor>
```

Comment:
It's not 'General', it's 'Generic', different meanings

Reply:
Fixed